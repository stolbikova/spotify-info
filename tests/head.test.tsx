import Header from "../components/Head";
import "@testing-library/jest-dom";
import { render, waitFor } from "@testing-library/react";


describe("Head", () => {
    it("renders a head", async () => {
      render(<Header />)
      await waitFor(() => {
        expect(document.querySelector('head')).toMatchSnapshot()
      })
    });
  });