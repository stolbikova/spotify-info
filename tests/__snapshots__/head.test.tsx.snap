// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head renders a head 1`] = `<head />`;
